config/manager.go

```diff
@@ -104,14 +108,27 @@ func LoadLists(isReload bool) bool {
 		}
 	}
 	// log.Println("Lists:", Config.Lists)
+	ListsModeTable := findListsMode()
 	if l := len(Config.Lists); l == 0 { // if nothing in Lists
 		Lists = map[string]*set.StringSet{} // empty map
 	} else {
 		Lists = make(map[string]*set.StringSet, l) // map size init
 		for k, v := range Config.Lists {
 			// log.Println("List: Loading", k, "value:", v)
-			list := set.NewStringSetFromSlice(v)
-			Lists[k] = &list
+			switch ListsModeTable[k] {
+			case "HypixelGuild":
+				// v[0] A Player Name in Guild
+				// v[1] Hypixel ApiKey
```

Comment:
This design is..... not graceful enough.

My opinion is add some new options to NameAccess object.

Btw, the new HypixelGuild mode is LGTM.